catalog/generators/rest-aspnetcore/index.ts

```diff
@@ -0,0 +1,33 @@
+
+import { BaseGenerator } from 'core/catalog/types';
+
+import PlatformDotNet, { PlatformDotNetProps } from 'generators/platform-dotnet';
+
+export interface RestAspNetCoreProps extends PlatformDotNetProps {
+}
+
+export default class RestAspNetCore extends BaseGenerator {
+    public static readonly sourceDir: string = __dirname;
+
+    public async apply(resources, props: RestAspNetCoreProps, extra: any = {}) {
+        // Check if the generator was already applied, so we don't do it twice
+        if (!await this.filesCopied()) {
+            // First copy the files from the base DoNet platform module
+            // and then copy our own over that
+            const pprops = {
+                'application': props.application,
+                'subFolderName': props.subFolderName,
+                'serviceName': props.serviceName,
+                'routeName': props.routeName,
+                'env': {
+                    'DOTNET_SDK_VERSION': '2.1.500'
```

Comment:
This is a workaround for https://github.com/redhat-developer/s2i-dotnetcore/issues/263

Reply:
No longer needed because 1. bug was resolved 2. we are not using the centos images anymore.